changelog.txt

```diff
@@ -6,3 +6,4 @@
 * Fixed bug where CORS headers were too restrictive.
 * Fixed bug where environment variables are unset for script in `emulators:exec`.
 * `emulators:exec` script now inherits stdout and stderr.
+* **BREAKING** The CLI no longer suppport being run in Node 6 environments.
```

Comment:
Move this to top.

Reply:
Sure. I'll make sure it stays there as well (as things are merged and updated, hopefully it stays in the right place)